### PR TITLE
Added VoiceEvent method get_guild

### PR DESCRIPTION
**Pull request by NobodyForYouCreator**
### Summary
In order to optimize all the events, it would be a good idea to add every event get_guild to make user's work as optimized as it can be

### Checklist

- [x] I have run `nox` and all the pipelines have passed.
- [x] I have made unittests according to the code I have added/modified/deleted.

### CODE REVIEW

**Comment by NobodyForYouCreator** [comment]
Created file in changelog

File: hikari/events/voice_events.py [hunk 1]
```diff
@@ -62,6 +63,21 @@ def guild_id(self) -> snowflakes.Snowflake:
         hikari.snowflakes.Snowflake
             The guild ID of the guild this event relates to.
         """
+    
+    def get_guild(self) -> typing.Optional[guilds.GatewayGuild]:
```
Comment (FasterSpeeding) [inline hikari/events/voice_events.py:67]: Could we get two test cases for this? It should be pretty simple to do, you can pretty much just copy something like these two tests https://github.com/hikari-py/hikari/blob/fff9bca164271dad51b8a62aedafd64f52ed4990/tests/hikari/events/test_message_events.py#L207-L215

Comment (FasterSpeeding) [inline hikari/events/voice_events.py:67]: If you want to, you could also add a similar fetch_guild method here

File: changes/891.feature.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+To make it possible to use events with decorators, where get_guild is required
```
Comment (FasterSpeeding) [inline changes/891.feature.md:1]: could we just reword this to something like "Added a get_guild (and possibly fetch_guild) shortcut method(s) to the voice events"

File: hikari/events/voice_events.py (hunk 1 above)
Comment (FasterSpeeding) [reply hikari/events/voice_events.py:67]: I'll do the fetch_guild method in another pr

**Comment by davfsa** [comment]
Closing due to inactivity and I dont think we will keep these methods around for long anyways
